quipucords/api/scantask/model.py

```diff
@@ -371,16 +376,21 @@ def cleanup_facts(self, identity_key):
         """
         if self.scan_type != ScanTask.SCAN_TYPE_INSPECT:
             return
+        # TODO: should we even be doing this cleanup without logging?
+        # my gut feeling is just mark it as failed (and stop post processing
+        # failed data)
```

Comment:
Now that you mention it, yeah, I think this could benefit from at least a `logger.warning` message here. That might help us to identify this "cleanup" operation when investigating future data anomalies.

Reply:
And even better: If we start seeing the new warning a lot in our local development and testing, that might indicate that we should follow your gut and abort processing more aggressively.